plugins/repository-gcs/src/main/java/org/opensearch/repositories/gcs/GoogleCloudStorageService.java

```diff
@@ -70,6 +71,15 @@ public class GoogleCloudStorageService {
      */
     private volatile Map<String, Storage> clientCache = emptyMap();
 
+    final private GoogleApplicationDefaultCredentials googleApplicationDefaultCredentials;
```

Comment:
Are those always "default" or just "credentials", in which case this should just be `credentials`.

Reply:
Yes, those would be always "default" because it refers to the Google's Application Default Credentials. 
Also, if we want to make it short I can suggest using GoogleADC instead of GoogleApplicationDefaultCredentials?

Reply:
I don't have a preference on length, I just want things to be semantically correct.

Reply:
I will change the name of the method as you suggested in the above comment.

Reply:
I am not sure whether I suppose to resolve the conversation, but since i have implemented the change as per suggestion closing the conversation. Thank you @dblock and please open a new one if you find anything else.